src/main/java/net/smartcosmos/cluster/auth/AuthServerApplication.java

```diff
@@ -59,6 +62,7 @@
 @SessionAttributes("authorizationRequest")
 @Slf4j
 @EnableDiscoveryClient
+@EnableSmartCosmosMonitoring
 public class AuthServerApplication extends WebMvcConfigurerAdapter {
```

Comment:
Take the opportunity to clean up the startup class and remove the `extends WebMvcConfigurerAdapter`?